### PR TITLE
DO-1076 Temp fix for older macos versions

### PR DESCRIPTION
**Pull request by mmaharjan-ccdc**

### CODE REVIEW

**Review by siong-chin — APPROVED** [review]
If brew can't be installed, when will happen to the following tasks?

File: tasks/Darwin.yml [hunk 1]
```diff
@@ -53,8 +53,16 @@
     state: absent
 
 - name: Install homebrew
-  ansible.builtin.include_role:
-    name: geerlingguy.mac.homebrew
+  block:
+    - include_role:
```
Comment (florianpiesche) [inline tasks/Darwin.yml:57]: should this be `ansible.builtin.include_role`?

**Review by florianpiesche — APPROVED** [review]

File: tasks/Darwin.yml (hunk 1 above)
Comment (mmaharjan-ccdc) [reply tasks/Darwin.yml]: updated, thanks

**Comment by mmaharjan-ccdc** [comment]
@siong-chin the ignore error was added by Jordan after various troubleshooting, I'm not too sure why but the logs show no issues with this. If it were to fail I assume the following steps would fail anyways.

**Pull request merged**